check_increasing_timestamp.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
Put into a util or bin directory in CFC_DataCollector to avoid cluttering up the main directory?